Makefile

```diff
@@ -105,8 +93,33 @@ test_cov: cov            ## Build and run unit tests with code coverage enabled.
 
 .PHONY: clean
 clean:                   ## Clear build/ directory.
-	rm -rf build/*
+	rm -rf ${DEBUG_BUILD_DIR}/*
+
+# ================= distribution part =================
+.PHONY: release
+release:                 ## Build vsag with release options.
+	cmake ${VSAG_CMAKE_ARGS} -B${RELEASE_BUILD_DIR} -DCMAKE_BUILD_TYPE=Release
+	cmake --build ${RELEASE_BUILD_DIR} --parallel ${COMPILE_JOBS}
+
+.PHONY: distribution
+distribution:            ## Build vsag with distribution options.
+	cmake ${VSAG_CMAKE_ARGS} -B${RELEASE_BUILD_DIR} -DCMAKE_BUILD_TYPE=Release -DENABLE_CXX11_ABI=off -DENABLE_LIBCXX=off
+	cmake --build ${RELEASE_BUILD_DIR} --parallel ${COMPILE_JOBS}
+
+.PHONY: libcxx
+libcxx:                  ## Build vsag using libc++.
+	cmake ${VSAG_CMAKE_ARGS} -B${RELEASE_BUILD_DIR} -DCMAKE_BUILD_TYPE=Release -DENABLE_LIBCXX=on
+	cmake --build ${RELEASE_BUILD_DIR} --parallel ${COMPILE_JOBS}
 
 .PHONY: install
 install:                 ## Build and install the release version of vsag.
-	cmake --install build/
+	cmake --install ${RELEASE_BUILD_DIR}/
+
+
+PARAM1 := "-DNUM_BUILDING_JOBS=${COMPILE_JOBS} -DENABLE_PYBINDS=1 -S. -B${RELEASE_BUILD_DIR} -DCMAKE_BUILD_TYPE=Release"
+PARAM2 := "--build ${RELEASE_BUILD_DIR} --parallel ${COMPILE_JOBS}"
```

Comment:
why use these parameters, rather than place them in L125

Reply:
These three parameters have different uses in the pyvsag build script. Write them separately and wrap them with quotas to make it clear.